src/xdocs/config_naming.xml

```diff
@@ -608,7 +608,7 @@
       <subsection name="Description">
         <p>
           Validates identifiers for local, <code>final</code> variables, including
-          <code>catch</code> parameters.
+          <code>catch</code> and <code>try-with-resource</code> parameters.
```

Comment:
`try-with-resource` isn't a parameter. `catch` is the parameter.
change to `<code>catch</code> parameters and resources in <code>try</code> statements.

Please also make similar change to top of `LocalFinalVariableNameCheck`'s class javadoc.